components/v1/Buttons/ConnectWalletDesktop.tsx

```diff
@@ -1,10 +1,10 @@
 import Link from "next/link";
+import { useRouter } from "next/router";
```

Comment:
@math-marcellino why we need to use `next/router` ?

Reply:
We use it detect the `pathname` sir. So if the `pathname` contain `binance`, then the `default chain` for that page will be BSC. Implementation can be seen in code below:
```
const selectedChain = router.pathname.includes("binance") ? customChains.bsc : Chains.arbitrumOne;
```

Reply:
@math-marcellino gotcha.

The problem is that if we have more chains, this line will be too long

```
const selectedChain = router.pathname.includes("binance") ? customChains.bsc : Chains.arbitrumOne;
```

but it's nice problem to have! let's just use this for now and revisit it later